static/templates/actors/familiar/sheet.hbs

```diff
@@ -0,0 +1,265 @@
+<form class="{{cssClass}}" autocomplete="off" spellcheck="false" data-tooltip-class="pf2e">
+    {{!-- HEADER --}}
+    <header>
+        <div class="image-container">
+            <img class="actor-image" src="{{actor.img}}" data-edit="img" />
+            <a class="hover-icon" data-action="show-image" data-tooltip="SIDEBAR.CharArt"><i class="fa-solid fa-image fa-fw"></i></a>
+        </div>
+
+        <div class="header-content">
+            <div class="charname">
+                <input class=" charname-value" name="name" type="text" value="{{actor.name}}"
+                    placeholder="{{localize "PF2E.NPC.Familiar"}}" spellcheck="false" />
+                <label class="familiar-title">{{localize "TYPES.Actor.familiar"}}</label>
+                {{#if data.master.id}}
+                    <label class="familiar-title"> {{data.details.level.value}}</label>
+                {{/if}}
+            </div>
+            <div class="sub-header">
+                <div class="traits">
+                    {{#each traits as |trait|}}
+                        <span data-trait="{{trait.value}}" data-tooltip="{{trait.description}}">{{localize trait.label}}</span>
+                    {{/each}}
+                    <span>{{localize size}}</span>
+                </div>
+                <div>
+                    <label for="{{document.uuid}}-master"><strong>{{localize "PF2E.Familiar.Master"}}</strong></label>
+                    <select id="{{document.uuid}}-master" name="system.master.id">
+                        {{#select master.id}}
+                            <option value="">{{localize "PF2E.Familiar.SelectMaster"}}</option>
+                            {{#if limitedSheet}}
+                                {{#if master}}<option value="{{master.id}}" selected>{{master.name}}</option>{{/if}}
+                            {{else}}
+                                {{#each masters as |eligible idx|}}
+                                    <option value="{{eligible.id}}">{{eligible.name}}</option>
+                                {{/each}}
+                            {{/if}}
+                        {{/select}}
+                    </select>
+                </div>
+            </div>
+            <section class="details-section">
+                {{> headerDetails }}
+            </section>
+        </div>
+    </header>
+    <hr>
+    <div class="main-section">
+        <div class="skills-sidebar {{#if limitedSheet}}skills-sidebar-placeholder{{/if}}">
+            {{> skills }}
+        </div>
+        <div class="main-container">
+            {{> details }}
```

Comment:
Put the `unless` here and save some indentation below

Reply:
Good call, fixed.